migrator/README.md

```diff
@@ -272,8 +275,12 @@ In migrator, there are a multiple ways to access data.
 3. Duplicate the Postgres Store
    This method is used in version 73 and 74 to migrate all tables from RocksDB to Postgres. In addition to frozen schema,
    the store to access the data are also frozen for migration. The migrations with this method are closely associated
-   with current release eg. search/delete with schema and the prototypes of the objects. This method is NOT recommended for
-   4.0 and beyond.
+   with current release eg. search/delete with schema and the prototypes of the objects. 
+
+   Now that stores are built upon a generic store you would need to make a copy of the store and remove
+   references to SAC and such.  We should be careful to not relying on the search framework for queries as those
```

Comment:
Should the copied stores also refer to frozen schemas instead of the latest schemas?

Reply:
done

Reply:
excellent point.